deploy/env/dev.yaml

```diff
@@ -11,4 +11,13 @@ registration-service:
                   "resource": "crt",
                   "clientId": "crt",
                   "public-client": true
-                }'
\ No newline at end of file
+                }'
+  verification.enabled: 'true'
+  verification.daily_limit: '3'
+  verification.attempts_allowed: '5'
+host-operator:
+  registration-service-url: 'https://sso.prod-preview.openshift.io/auth/realms/toolchain-public/protocol/openid-connect/certs'
+  console-namespace: "openshift-console"
+  console-route-name: "console"
+  che-namespace: "toolchain-che"
+  che-route-name: "che"
```

Comment:
Those are the default values we use in configuration.go, right? So, why adding them here?

Reply:
we seem to have a few things added here that were already part of defaults in configuration.go such as the `config-raw` for instance. so i wasn't too sure if there was a reason for this and just followed what was already there. I will remove all the values that have defaults from here in that case

Reply:
removed fields that had defaults here: b03cfdd25a240c85b66225bceb6bdce25f774c59 and here: 0bd59fa6669e3cf716c0411dd11adc2b9261ba8b